Dockerfiles/entrypoints/tldw-app-first-run.sh

```
@@ -103,20 +108,93 @@ if [ "$AUTH_MODE" = "single_user" ]; then
   upsert_env "SINGLE_USER_API_KEY" "$current_key"
 fi
 
+# Auto-generate MCP secrets if missing or placeholder (min 32 chars each)
+for mcp_var in MCP_JWT_SECRET MCP_API_KEY_SALT BYOK_ENCRYPTION_KEY; do
+  eval current_val="\${$mcp_var:-}"
+  case "$current_val" in
```

Shellcheck (0.11.0)

[warning] current_val is referenced but not assigned.

(SC2154)